index.md

```diff
@@ -8,30 +8,30 @@ As also described in the [PSR Workflow Bylaw][workflow]. The Editor, or editors,
 
 ### Accepted
 
-| Num | Title                          | Editor                         |  Coordinator            | Sponsor                 |
```

Comment:
Isn't that historical info we want to keep?

Reply:
I will have to make another pass, but the intent is to ensure all information is available in the meta document. I introduced meta documents for PSR's that don't have it.